doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -216,75 +171,69 @@ simply :g:`t=u` dropping the implicit type of :g:`t` and :g:`u`.
       dependencies in the goal.
```

Comment:
In the error messages above, the `:n:` should be removed since the notation syntax is already activated by default (as in commands).